src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,149 @@
+.contacts {
+  margin-top: 80px;
+
+  @include onTablet {
+    margin-top: 120px;
+  }
+
+  @include onDesktop {
+    margin-top: 180px;
+  }
+}
+
+.content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 24px;
+  row-gap: 48px;
+
+  @include onTablet {
+    grid-template-columns: repeat(6, 1fr);
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  &__form {
+    grid-column: 1/-1;
+
+    @include onTablet {
+      grid-column: 1/4;
+    }
+
+    @include onDesktop {
+      grid-column: 1/6;
+    }
+  }
+
+  &__info {
+    grid-column: 1/-1;
+
+    @include onTablet {
+      grid-column: 4/-1;
+    }
+
+    @include onDesktop {
+      grid-column: 8/11;
+    }
+  }
+}
+
+.form {
+  color: #7e7e83;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+
+  & > * {
+    color: #292929;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+  }
+
+  & > * ::placeholder {
+    color: #7e7e83;
+  }
+
+  &__name {
+    display: block;
+    background-color: #f8f8fa;
+
+    padding: 14px 16px;
+    width: 100%;
+    height: 48px;
+    border-radius: 40px;
+    margin-bottom: 16px;
+
+    &:hover {
+      border: 1px solid #d5d7de;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new `border` to the element on hover. Add default transparent border of the same width, and change its color on `:hover`